CONTRIBUTING.md

```diff
@@ -6,9 +6,31 @@ This project and everyone participating in it is governed by the istudybucket Co
 If you want to ask a question, we assume that you have read the available Documentation.
 
 Before you ask a question, it is best to search for existing Issues that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
-Also Endevour to follow our [Code of Conduct](https://github.com/istudybucket/istudybucket-backend/blob/master/CODE_OF_CONDUCT.md)
 
-## CONTRIBUTING
+# Contributing to istudybucket
+
+A big welcome and thank you for considering contributing to Auth0 open source projects! It’s people like you that make it a reality for users in our community.
```

Comment:
Instead of: 

> A big welcome and thank you for considering contributing to Auth0 open source projects!

Could be:

> A big welcome and thank you for considering contributing to iStudyBucket!